packages/@react-aria/dnd/src/useDroppableCollection.ts

```diff
@@ -21,7 +21,8 @@ import {useAutoScroll} from './useAutoScroll';
 import {useDrop} from './useDrop';
 import {useDroppableCollectionId} from './utils';
 
-export interface DroppableCollectionOptions extends DroppableCollectionProps {
+// TODO omit getDropOperation since it isn't used here in useDroppableCollection? Find any other props that aren't used and omit for clarity (onDropEnter, onDropMove, onDropExit)
+export interface DroppableCollectionOptions extends Omit<DroppableCollectionProps, 'getDropOperation'> {
```

Comment:
I'll go through some of the other dnd hooks and clean out any props that are allowed but not actually used in the hooks